src/meltano/cli/interactive/config.py

```diff
@@ -0,0 +1,408 @@
+"""Interactive configuration handler."""
+
+from __future__ import annotations
+
+import contextlib
+import json
+
+import click
+from jinja2 import BaseLoader, Environment
+from rich.console import Console, Group
+from rich.markdown import Markdown
+from rich.panel import Panel
+from rich.table import Table
+from rich.text import Text
+
+from meltano.cli.interactive.utils import InteractionStatus
+from meltano.cli.utils import CliError
+from meltano.core.environment_service import EnvironmentService
+from meltano.core.settings_service import REDACTED_VALUE, SettingKind, SettingValueStore
+from meltano.core.settings_store import StoreNotSupportedError
+
+PLUGIN_COLOR = "magenta"
+ENVIRONMENT_COLOR = "orange1"
+SETTING_COLOR = "blue1"
+VALUE_COLOR = "green"
+
+HOME_SCREEN_TEMPLATE = """[bold underline]Configuring [{{ plugin_color }}]{{ plugin_name.capitalize() | safe }}[/{{ plugin_color }}] {% if environment_name %}in Environment[{{ environment_color }}]{{ environment_name }}[/{{ environment_color }}] {% endif %}Interactively[/bold underline]
```

Comment:
This seems ripe for a typo in the templating in future changes , would be good to get a quicky test of this template generation in a follow up.

Reply:
Created #6735 to track.